frameworks/digital-outcomes-and-specialists-4/questions/briefs/evaluationTypeSpecialists.yml

```diff
@@ -15,22 +15,22 @@ optional: true
 options:
   - label: "Reference"
     description: >
-       A written reference is a good way of confirming whether a specialist has done the work they’ve said they’ve
-       done. You should always ask for references that relate to a specific timeframe, eg work they’ve done in the
-       last 2 years.
+      A written reference is a good way of confirming whether a specialist has done the work they’ve said they’ve
+      done. You should always ask for references that relate to a specific timeframe, for example: work they’ve done
+      in the last 2 years.
```

Comment:
Please replace 'for example: with: such as

Reply:
Done

Reply:
Done